docs/howto/operate/index.md

```diff
@@ -14,4 +14,5 @@ delete-hub.md
 manual-deploy.md
 test-hub.md
 grafana.md
+node-adm.md
```

Comment:
Can we keep the full words in the file name, rather than short-hand? (so, `administration` rather than `adm`). It's nice to have shorter URLs but I think it's more important that things are human-glanceable